packages/@okta/vuepress-site/docs/reference/token-hook/index.md

```diff
@@ -660,3 +664,45 @@ You then need to associate the registered Inline Hook with a Custom Authorizatio
 
 > **Note:** Only one Inline Hook can be associated with each rule.
 
+## Troubleshooting
+
+This section covers what happens when a token inline hook flow fails either due to the external inline hook service returning an error object or not returning a successful response, or the inline hook patch fails.
+
+### When there is an error
+
+- When a token inline hook failure is related to an error that occurs when performing an inline hook PATCH command (for example, updating, adding, and deleting claims), the inline hook operation is skipped. The token is generated without any modification from the inline hook.
```

Comment:
When a hook command (for example, updating, adding, and deleting claims) cannot be performed,  the inline hook operation is skipped. The token is generated without any modification from the inline hook.